app/controllers/users_controller.rb

```diff
@@ -1,5 +1,15 @@
 class UsersController < ApplicationController
+
   def show
     @user = User.find(params[:id])
   end
+
+  def update
+    if @user.update(user_params)
+      @user.profile_photo = "http://sellleadsucceed.files.wordpress.com/2014/02/bigstock-silhouette-with-a-question-mar-59367497.jpg"
```

Comment:
This line doesn't do anything since you're not saving the model (the `update` call will save).